pkg/rules/validation.go

```diff
@@ -61,7 +65,8 @@ func (p *PayloadValidator) Validate(ctx context.Context, method, endpoint string
 	if p.isCreateOrUpdateRule(method, endpoint) {
 		ruleRequest := new(oathkeeper.Rule)
 		if err := json.Unmarshal(body, ruleRequest); err != nil {
-			return ctx, nil, err
+			p.logger.Info("Json parsing error: ", err)
+			return ctx, nil, fmt.Errorf("failed to parse JSON body")
```

Comment:
I removed it because I thought that most of the time the go error is not user friendly, so it isn't very helpful to expose to the user. No problem with adding it back

Reply:
that's ok as it is then, thanks